blueprints/ember-cli-nouislider/index.js

```diff
@@ -2,6 +2,6 @@ module.exports = {
   normalizeEntityName: function() {}, // no-op since we're just adding dependencies
 
   afterInstall: function(options) {
-    return this.addBowerPackageToProject('nouislider'); // is a promise
+    return this.addBowerPackageToProject('nouislider', '^8.0.1'); // is a promise
```

Comment:
I locked this down to version 8 so this sort of problem doesn't suddenly come up again.